ui/app/selectors/permissions.js

```diff
@@ -192,5 +192,5 @@ function getAccountsCaveatFromPermission (accountsPermission = {}) {
 }
 
 function domainSelector (state, origin) {
-  return origin && state.metamask?.domains[origin]
+  return origin && state.metamask?.domains?.[origin]
```

Comment:
Nit: `metamask` is guaranteed to always be an object - the `?.` was put in the wrong spot originally.

```suggestion
  return origin && state.metamask.domains?.[origin]
```